adapter/config/default_config.go

```diff
@@ -134,6 +139,7 @@ var defaultConfig = &Config{
 			AllowPartialMessage: false,
 			PackAsBytes:         false,
 		},
+		MTLSAPIsEnabled: false,
```

Comment:
Since this is only related to Downstream,  what if we move this inside `Downstream`?